src/components/Checkbox/Checkbox.css

```diff
@@ -0,0 +1,25 @@
+.checkbox {
+    width: 100vw;
+    height: 24px;
+    display: flex;
+    align-items: center;
+    margin: 24px;
+    font-size: 16px;
```

Comment:
This could be applied to the `checkbox__text` class, it would be more specific that way